src/setting/setting.service.ts

```diff
@@ -0,0 +1,40 @@
+import { SettingSchema } from '@/schema/setting.schema';
+import { SettingDto } from '@/setting/dto/setting.dto';
+import { Injectable } from '@nestjs/common';
+import { ReturnModelType } from '@typegoose/typegoose';
+import { InjectModel } from 'nest-typegoose';
+
+@Injectable()
+export class SettingService {
+  constructor(
+    @InjectModel(SettingSchema)
+    private readonly settingModel: ReturnModelType<typeof SettingSchema>,
+  ) {}
+
+  async getSettings() {
+    const settings = await this.settingModel.find()[0];
```

Comment:
You can use findOne without condition to get same result.
Don't forget to .exec()